cranelift/codegen/src/isa/x86/unwind.rs

```diff
@@ -215,11 +336,40 @@ impl UnwindInfo {
             return None;
         }
 
+        if saved_fpr {
+            if static_frame_allocation_size > 240 && saved_fpr {
+                panic!("stack frame is too large ({} bytes) to use with Windows x64 SEH when preserving FPRs. \
```

Comment:
We have a codegen error for implementation limits. Can we return a Result from this function, and return the impl limit reached error in this case, instead of panicking?

(Otherwise, this blocks fuzzing, among other things.)

Reply:
I'm curious for @peterhuene's thoughts on this, because #1466 will hopefully land relatively quickly after this does. Does #1466 PR introduce a reason for unwind information to be fallible?

If this is the only reason for an occasional failure, we could make this `Result` with the understanding that when it stops being `Result` we can revert this back to a non-Result function. I'm not sure that there's a reason this should be fallible, generally speaking. I only prefer a panic because I see this as something that _should_ be an important issue to resolve promptly, but I realize that might not be very nice to downstream users (including us fuzzing :) ).

Reply:
Sorry, I missed this notification yesterday.  I think making it `Result` is a good idea even if this particular platform's implementation will hopefully be infallible once this restriction is lifted.  Other unwind information implementations might be fallible and it can't hurt to have some signature consistency between the functions that create the unwind info.

Reply:
I've made functions for unwind emission all `Result`-y. `ImplLimitExceeded` doesn't describe _which_ limit was exceeded, but I figure for the cases in Windows unwind info it would be very helpful to get some kind of hint, so I moved the panic message into a `log::warn` as well.